app/controllers/users_controller.rb

```diff
@@ -4,6 +4,6 @@ def index
   end
 
   def show
-    @user = User.find(params[:id])
```

Comment:
- [ ] _This is intended for posts controller index, writing it here since was not changed_:
Please fix the n+1 problem by fetching all the comments for a given post like this @posts = `@user.posts.includes(:comments)`

Reply:
This hs already been resolved